lib/shopify_api/resources/api_checkout.rb

```diff
@@ -0,0 +1,32 @@
+module ShopifyAPI
+  class CheckoutAPI < Base
+    self.prefix = '/api/'
```

Comment:
We could remove this and use the proper admin endpoints now.

Payments is not 100% there just yet, but we can make it point to `/admin/applications/checkouts/#{checkout_id}/payments.json` until we are fully able to move everything.